common/presenters/message-banner/assessment-to-handed-over-banner.js

```diff
@@ -33,6 +33,17 @@ module.exports = function assessmentToHandedOverBanner({
           context,
         })}
       </a>
+
+      <p class="app-icon app-print--hide">
```

Comment:
You can copy the markup from [this change](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1321/files#diff-07a3ade12b6c6f2871d31e04cb06e5f4c9c1ea66bbd06259216c803224d43500R57) to add this I think. In fact, we could just bundle into that change...